src/fides/api/ops/service/messaging/message_dispatch_service.py

```diff
@@ -117,12 +118,14 @@ def dispatch_message(
     message_body_params: Optional[
         Union[
             AccessRequestCompleteBodyParams,
+            ConsentEmailFulfillmentBodyParams,
             SubjectIdentityVerificationBodyParams,
             RequestReceiptBodyParams,
             RequestReviewDenyBodyParams,
             List[CheckpointActionRequired],
         ]
     ] = None,
+    subject_override: Optional[str] = None,
```

Comment:
I added this subject_override parameter for cases where the subject needs dynamic content.  It's only applicable for emails right now.